source/renderer/app/components/voting/voting-info/CurrentFund.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import {
+  VOTING_REGISTRATION_CAST_END_DATE,
+  CURRENT_VOTING_FUND_NUMBER,
+} from '../../../config/votingConfig';
+import {
+  formattedDateTime,
+  mapToLongDateTimeFormat,
+} from '../../../utils/formatters';
+import type { Locale } from '../../../../../common/types/locales.types';
+import { ExternalLinkButton } from '../../widgets/ExternalLinkButton';
+import type { Intl } from '../../../types/i18nTypes';
+import styles from './CurrentFund.scss';
+import { messages } from './CurrentFund.messages';
+
+type Props = {
+  currentLocale: Locale,
+  currentDateFormat: string,
+  currentTimeFormat: string,
+  onExternalLinkClick: Function,
+  intl: Intl,
```

Comment:
We can use the built-in `import { intlShape } from 'react-intl';` for this type, so it keeps consistent with other components using it